bin/check-package-metadata.js

```diff
@@ -0,0 +1,149 @@
+      );
+    }
+  }
+}
+
+async function checkCodeOwnersFile(packages, rootPath) {
+  const excludes = ['@loopback/sandbox-example'];
+
+  const codeOwnersFile = await fs.readFile('CODEOWNERS', 'utf8');
+
+  for (const p of packages) {
+    if (excludes.includes(p.name)) {
+      continue;
+    }
+
+    const packagePath = path.relative(rootPath, p.location);
+
+    if (!codeOwnersFile.includes(packagePath)) {
+      throw new Error(
+        `CODEOWNERS.md doesn't contain mention about the ${p.name}`,
+      );
+    }
+  }
+}
+
+async function checkPkgsPackageJson(packages, rootPkg) {
+  const pkgs = packages.filter(p => !p.private);
+
+  function getCopyrightOwner(pkg) {
+    return pkg['copyright.owner'] || (pkg.copyright && pkg.copyright.owner);
+  }
+
+  function wrongFieldValueError(packageName, field, original) {
+    throw new Error(
+      `${packageName} package.json has incorrect ${field} field value. It must be ${original}`,
+    );
+  }
+
+  for (const p of pkgs) {
+    const packageName = p.name;
+    const pkg = fs.readJsonSync(path.join(p.location, 'package.json'));
+
+    const isPublicAccess =
+      pkg.publishConfig &&
+      pkg.publishConfig.access &&
+      pkg.publishConfig.access === 'public';
+
+    if (!isPublicAccess) {
+      throw new Error(
+        `${p.name} package.json doesn't contain publicConfig.access equals public`,
+      );
+    }
+
+    if (pkg.author !== rootPkg.author) {
+      wrongFieldValueError(packageName, 'author', rootPkg.author);
+    }
+
+    if (getCopyrightOwner(pkg) !== getCopyrightOwner(rootPkg)) {
+      wrongFieldValueError(
+        packageName,
+        'copyright.owner',
+        getCopyrightOwner(rootPkg),
+      );
+    }
+
+    if (pkg.license !== rootPkg.license) {
+      wrongFieldValueError(packageName, 'license', rootPkg.license);
+    }
+
+    const isCorrectRepositoryUrl =
+      pkg.repository &&
+      pkg.repository.url &&
+      pkg.repository.url.includes(
+        'https://github.com/strongloop/loopback-next',
+      );
+
+    if (!isCorrectRepositoryUrl) {
+      wrongFieldValueError(
+        packageName,
+        'repository.url',
+        rootPkg.repository.url,
+      );
+    }
+
+    const isCorrectRepositoryDirectory =
+      pkg.repository && pkg.repository.directory && fs.existsSync(p.location);
+
+    if (!isCorrectRepositoryDirectory) {
+      throw new Error(`${packageName} directory doesn't exist in monorepo`);
+    }
+  }
+}
+
+async function checkPackagesMetadata() {
+  const project = new Project(process.cwd());
+  const packages = await project.getPackages();
+  const rootPath = project.rootPath;
+  const rootPkg = fs.readJsonSync('package.json');
+
+  await checkLicenseFiles(packages);
+  await checkMonorepoFile(packages, rootPath);
+  await checkCodeOwnersFile(packages, rootPath);
+  await checkPkgsPackageJson(packages, rootPkg);
+}
+
+if (require.main === module) {
+  checkPackagesMetadata().catch(err => {
+    console.error(err);
```

Comment:
Please explain the users how they can re-run the check locally to verify that all problems have been fixed. 

Example message:

```
Some of the package are not following loopback-next guidelines:

- @loopback/some-name directory doesn't exist in monorepo
- (...more messages...)

Please fix the problems listed above and run the following command to verify:
  npm run check-package-metadata
```

Reply:
@bajtos Thank you for feedback. Do you offer to gather all the information and return list of all problem packages? Because I chose another way, the script throws the error on each step, which will be encountered, while the user will not fix all problem places. If you consider your approach as more appropriate I will fix the script, but I need your final decision.

Reply:
@bajtos After some thought, I come to the conclusion that you are right. Otherwise the user should run script again and again instead of get the full list one time, it's more user-friendly approach. I'will fix it